packages/components/src/components/video/video.ts

```diff
@@ -0,0 +1,137 @@
+ * loading appearance of wrapped video player.
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart play-button - Button element wrapper around the play-icon slot.
+ */
+
+@customElement('sd-video')
+export default class SdVideo extends SolidElement {
+  /** Set to `true` to hide the play icon and the overlay. */
+  @property({ type: Boolean, reflect: true }) playing = false;
+
+  /** Set to `true` to show a dark overlay. Only used when `playing` is `false`. */
+  @property({ type: Boolean, reflect: true }) overlay = false;
+
+  private readonly hasSlotController = new HasSlotController(this, '[default]', 'play-icon', 'poster');
+
+  /** Getter for optional poster slot. */
+  private get poster(): Element | null {
+    const slot: HTMLSlotElement = this.shadowRoot!.querySelector('slot[name=poster]')!;
+
+    if (slot?.assignedElements().length > 0) {
+      return slot.assignedElements()[0];
+    }
+    return null;
+  }
+
+  /** Method to fade out poster after initial play. */
+  private fadeoutPoster(): void {
+    if (this.poster instanceof HTMLImageElement) {
+      this.poster.style.opacity = '0';
+    }
+  }
+
+  /** Method to hide poster after initial play & fadeout. */
+  private hidePoster() {
+    if (this.poster instanceof HTMLImageElement) {
+      this.poster.style.display = 'none';
+    }
+  }
+
+  /** Internal method to group play behaviors. If a KeyboardEvent is used, refocus on the native video element to give the user seamless keyboard control. */
+  private play(e: MouseEvent | KeyboardEvent) {
+    this.emit('sd-play');
+    this.playing = true;
+    this.fadeoutPoster();
+
+    if (e instanceof KeyboardEvent) {
+      setTimeout(() => {
+        this.querySelector('video')?.focus();
+      });
+    }
+  }
+
+  render() {
+    return html`
+      <div part="base" aria-label="Video Player">
+        ${this.hasSlotController.test('poster')
+          ? html`<slot name="poster" part="poster" role="presentation" @transitionend=${this.hidePoster}></slot>`
+          : null}
+        <div
+          part="overlay"
+          id="overlay"
+          role="presentation"
+          class=${cx(
+            this.overlay && !this.playing ? 'opacity-100' : 'opacity-0',
+            'bg-[rgba(0,0,0,0.65)] w-full h-full absolute top-0 left-0 pointer-events-none z-20 play-pause-transition'
+          )}
+        ></div>
+        <button
+          part="play-button"
+          aria-label="Play video"
+          tabindex="0"
+          @click=${this.play}
+          @keydown=${this.play}
+          class=${cx(
+            this.playing ? 'opacity-0 pointer-events-none' : 'opacity-100',
+            'absolute left-[50%] top-[50%] translate-x-[-50%] translate-y-[-50%] p-4 bg-white bg-opacity-75 rounded-full flex items-center justify-center z-30 play-pause-transition'
```

Comment:
```suggestion
            'absolute left-1/2 top-1/2 translate-x-1/2 translate-y-1/2 p-4 bg-white bg-opacity-75 rounded-full flex items-center justify-center z-30 play-pause-transition'
```

Something like this should work.

Reply:
Before:
![image](https://github.com/solid-design-system/solid/assets/39494579/7cf3b09d-4a8c-4ac4-b429-f51fc5110e9a)

After:
![image](https://github.com/solid-design-system/solid/assets/39494579/58c9aa25-7333-4033-9978-f5b0ab025cbf)